### DISCUSSION TITLE
pydantic Field alias question

### DISCUSSION THREAD

**Discussion by panla** [fastapi/fastapi#8728] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class MapGroup(Model):
    group_id = fields.CharField(max_length=64)

    @property
    def group_id_(self):
        return self.group_id

class Entity(BaseModel):
    id: str = Field(..., alias='group_id_')
    class Config:
        orm_mode = True

@router.get('/orm-test', response_model=Entity)
async def orm_test():
    group = await MapGroup.get_or_none(id=1)
    logger.info(Entity.from_orm(group))
    return group
```

### Description

```text
from_orm:

    "id": "e3dea0f5-37f2-4d79-ae58-490af3228060"

api response:

{
    "group_id_": "e3dea0f5-37f2-4d79-ae58-490af3228060"
}

how can make them same?? 
```

### Operating System

Linux

### FastAPI Version

0.68.2

### Python Version

3.8.12

**Comment by harunyasar** [comment]
As far as I know, aliased field names are used for model inputs. In order to use aliased fields in dictionary representation of a model you could call `dict(by_alias=True)`. There is only one thing that comes to my mind right now and that is to override the `dict` method of the `BaseModel`.

```python
class Entity(BaseModel):
    id: str = Field(..., alias='group_id_')

    class Config:
        orm_mode = True

    def dict(
        self,
        *,
        include: Union['AbstractSetIntStr', 'MappingIntStrAny'] = None,
        exclude: Union['AbstractSetIntStr', 'MappingIntStrAny'] = None,
        by_alias: bool = False,
        skip_defaults: bool = None,
        exclude_unset: bool = False,
        exclude_defaults: bool = False,
        exclude_none: bool = False,
    ) -> 'DictStrAny':
        return super().dict(by_alias=True)
```

**Comment by artofhuman** [comment]
Now you can use param `response_model_by_alias=False`
```python
@router.get('/orm-test', response_model=Entity, response_model_by_alias=False)
async def orm_test():
```

Simple pydantic example
```python
    from pydantic import BaseModel, Field

    class MyModel(BaseModel):
        test: str = Field(alias="name")

        class Config:
            orm_mode = True

    inviter = f.ContractorFactory(name="Inviter")
    res = MyModel.from_orm(inviter)

res.dict()
#=> {'test': 'Inviter'}

res.dict(by_alias=True)
#=> {'name': 'Inviter'}
```

**Comment by YuriiMotov** [comment, marked as answer]
In current version `id: str = Field(..., validation_alias='group_id_')` should help
